pyramid_zipkin/tween.py

```diff
@@ -164,6 +171,11 @@ def tween(request):
 
         with zipkin_span(**tween_kwargs) as zipkin_context:
             response = handler(request)
+            if zipkin_settings.use_pattern_as_span_name and request.matched_route:
```

Comment:
rather than adding yet-another-flag, would it make sense to make it a hook that users can override?

Reply:
This is the diff implementing it as a hook: https://i.fluffy.cc/m5KKHmnCmG5RnXX1z2v3Wz6Z2GwH5D59.html

Imo that adds extra complexity and is more confusing for the user for little benefit. I don't think many people will want to override the span name post request with anything else.
And if they do we can change this afterwards.

Reply:
i see your point. im fine with discarding my suggestion.